### PR TITLE
Adds capability to selectively skip ddns updates using pillar

### PR DESCRIPTION
**Pull request by eemperor**

### CODE REVIEW

File: name-computer/map.jinja [hunk 1]
```diff
@@ -53,6 +55,7 @@ RedHat:
       - name: dnspython
       - reload_modules: True
 
+    {%- if not skip_forward_nsupdate == true %}
```
Comment (lorengordon) [inline name-computer/map.jinja]: These are boolean, so can simplify:

```suggestion
    {%- if not skip_forward_nsupdate %}
```

File: name-computer/map.jinja [hunk 2]
```diff
@@ -64,7 +67,9 @@ RedHat:
         - pip: Install dnspython
       - ttl: 7200
       - zone: {{ dns_domain }}
+    {%- endif %}
 
+    {%- if not skip_reverse_nsupdate == true %}
```
Comment (lorengordon) [inline name-computer/map.jinja]:
```suggestion
    {%- if not skip_reverse_nsupdate %}
```

File: name-computer/map.jinja [hunk 3]
```diff
@@ -19,6 +19,8 @@
 
 {%- set dns_domain = salt.pillar.get('name-computer:lookup:dns_domain') | default('', true) %}
 {%- set nameserver = salt.pillar.get('name-computer:lookup:nameserver') | default('', true) %}
+{%- set skip_forward_nsupdate = salt.pillar.get('name-computer:lookup:skip_forward_nsupdate') | default('', true) %}
+{%- set skip_reverse_nsupdate = salt.pillar.get('name-computer:lookup:skip_reverse_nsupdate') | default('', true) %}
```
Comment (lorengordon) [inline name-computer/map.jinja]: These are boolean, can default explicitly to false

```suggestion
{%- set skip_forward_nsupdate = salt.pillar.get('name-computer:lookup:skip_forward_nsupdate') | default(false, true) %}
{%- set skip_reverse_nsupdate = salt.pillar.get('name-computer:lookup:skip_reverse_nsupdate') | default(false, true) %}
```

**Review by lorengordon — CHANGES_REQUESTED** [review]
Can simplify the boolean logic a bit...

File: name-computer/map.jinja (hunk 1 above)
Comment (eemperor) [reply name-computer/map.jinja]: Ok.  Updated.

File: name-computer/map.jinja (hunk 2 above)
Comment (eemperor) [reply name-computer/map.jinja]: Ok.  Updated.

File: name-computer/map.jinja (hunk 3 above)
Comment (eemperor) [reply name-computer/map.jinja]: Ok.  Updated.

**Review by lorengordon — APPROVED** [review]

**Pull request merged**